datamanagement/query_gsc_for_dlp_fastqs.py

```diff
@@ -644,6 +672,15 @@ def main(storage_name,
     colossus_api = ColossusApi()
     tantalus_api = TantalusApi()
 
+    # load config file
```

Comment:
Does this kind of scoping work?  Even if it does I think you should be explicit and pass the config to import_gsc_dlp_paired_fastqs where its used, or even better, just load it closer to where its needed, its not going to take long.